src/semantics/denote.ts

```diff
@@ -351,10 +351,12 @@ const littleVAgent = λ('e', ['s'], c =>
 // λ𝘗. 𝘗
 const na = λ(['e', 't'], [], c => v(0, c));
 
-function denoteLittleV(toaq: string | null): Expr {
+function denoteLittleV(toaq: string | null): Expr | null {
```

Comment:
I think I'd prefer to split this up into `denoteCovertLittleV` and `denoteExplicitLittleV` functions now (the covert values aren't really Toaq…)